src/UI/Implementation/Component/Item/Renderer.php

```diff
@@ -108,7 +109,7 @@ protected function renderStandard(Item $component, RendererInterface $default_re
                 $tpl->setVariable("LEAD_IMAGE", $default_renderer->render($lead));
                 $tpl->parseCurrentBlock();
             }
-            if ($lead instanceof Icon) {
+            if ($lead instanceof Icon || $lead instanceof Avatar) {
                 $tpl->setCurrentBlock("lead_icon");
                 $tpl->setVariable("LEAD_ICON", $default_renderer->render($lead));
```

Comment:
lead_icon should be renamed to lead_icon_or_avatar or something more general here